Podfile.lock

```diff
@@ -1,12 +1,12 @@
 PODS:
-  - Mapbox-iOS-SDK (4.1.0)
+  - Mapbox-iOS-SDK (4.3.0)
   - MapboxCoreNavigation (0.17.0-beta.1):
     - MapboxDirections.swift (~> 0.20.0)
     - MapboxMobileEvents (~> 0.4)
     - Turf (~> 0.1)
   - MapboxDirections.swift (0.20.0):
     - Polyline (~> 4.2)
-  - MapboxMobileEvents (0.4.1)
+  - MapboxMobileEvents (0.4.51)
   - MapboxNavigation (0.17.0-beta.1):
```

Comment:
The builds are failing because navigation SDK v0.17.0-beta.1 is affected by mapbox/mapbox-navigation-ios#1607. You’ll need to also upgrade the navigation SDK dependency to v0.19.1 or above. Replace the `MapboxCoreNavigation` and `MapboxNavigation` entries in Podfile with this single entry:

```rb
pod 'MapboxNavigation', '~> 0.19.1'
```

Reply:
```
$ pod install
Analyzing dependencies
[!] CocoaPods could not find compatible versions for pod "Mapbox-iOS-SDK":
  In snapshot (Podfile.lock):
    Mapbox-iOS-SDK (= 4.3.0, ~> 4.0, ~> 4.3.0)

  In Podfile:
    Mapbox-iOS-SDK (~> 4.3.0)

    MapboxNavigation (~> 0.19.1) was resolved to 0.19.2, which depends on
      Mapbox-iOS-SDK (~> 4.2.0)
```

Reply:
MapboxNavigation 0.20.0 evidently fixes this incompatibility.